### PR TITLE
fix(FEC-10516): Fullscreen event dispatched twice from the player

### PR DESCRIPTION
**Pull request by dan-ziv**
### Description of the Changes

Add flag to overcome browsers which supports more than one fullscreenchange event.

### CheckLists

- [ ] changes have been done against master branch, and PR does not conflict
- [ ] new unit / functional tests have been added (whenever applicable)
- [ ] test are passing in local environment
- [ ] Travis tests are passing (or test results are not worse than on master branch :))
- [ ] Docs have been updated

### CODE REVIEW

File: src/fullscreen/fullscreen-controller.js [hunk 1]
```diff
@@ -19,6 +19,8 @@ class FullscreenController {
   _player: Player;
   _isInBrowserFullscreen: boolean;
   _eventManager: EventManager;
+  // Flag to overcome browsers which supports more than one fullscreenchange event
+  _isFullscreenEventDispatched: boolean = false;
```
Comment (Yuvalke) [inline src/fullscreen/fullscreen-controller.js:23]: Which browsers does it?
It was in the past and I removed it since shaka polyfill cause this issue.

**Review by Yuvalke — CHANGES_REQUESTED** [review]

**Review by OrenMe — APPROVED** [review]

File: src/fullscreen/fullscreen-controller.js (hunk 1 above)
Comment (dan-ziv) [reply src/fullscreen/fullscreen-controller.js:23]: Has nothing to do with Shaka AFAIK. 
Happens on Safari which supports `fullcreenchange` and `webkitfullscreenchange`
and on IE/Edge  which supports `fullcreenchange` and `msfullscreenchange`.

Comment (Yuvalke) [reply src/fullscreen/fullscreen-controller.js:23]: Shaka fixed this polyfill on their side

Comment (OrenMe) [reply src/fullscreen/fullscreen-controller.js:23]: these are double events we getting from registering them all.
We will open a task to register the correct events like screenfull library is doing.

**Pull request merged**